configuration/package_scripts/packages.json

```diff
@@ -12,29 +12,19 @@
                 // Don't stop or restart service when upgrading as the old agent
                 // is responsible for the entire OTA update (as the packages are installed by the agent)
                 // and restarting the service would kill the maintainer scripts mid-execution
-                {"name": "tedge-agent", "enable": false, "start": true, "restart_after_upgrade": false, "stop_on_upgrade": false}
+                {"name": "tedge-agent", "enable": false, "start": false, "restart_after_upgrade": false, "stop_on_upgrade": false}
```

Comment:
I'm assuming this change was done to avoid the agent starting with the wrong tedge config settings, esp on child devices, where quite a lot of settings like the topic id are expected to be updated, before starting it up. Or, did the merging of plugins also have any impact on this decision?

Reply:
Yes, being stopped by default means that it gives the user a chance to change the device_id topic.

For the main device it is also less of an issue as the tedge-agent is actually started and enabled using `tedge connect x`